mpt_extension_sdk/models/order.py

```
@@ -0,0 +1,104 @@
+        """Return the seller identifier when available."""
+        return None if self.seller is None else self.seller.id
+
+    @property
+    def downsize_lines(self) -> list[OrderLine]:
+        """Downsize lines from the order."""
+        return [elem for elem in self.lines if elem.quantity < elem.old_quantity]
+
+    @property
+    def upsize_lines(self) -> list[OrderLine]:
+        """Upsize lines from order."""
+        return [elem for elem in self.lines if elem.quantity >= elem.old_quantity > 0]  # noqa: WPS334
```

Ruff (0.15.10)

[warning] Invalid rule code in `# noqa`: WPS334

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)